### PR TITLE
Daniel task/ffmpeg

### PR DESCRIPTION
**Pull request by mukutdanielroy**
Fixes #

- [x] Does not depend on other PRs
- [ ] Depends on other PR being merged first: #

## What kind of change does this PR introduce (check at least one)
File compress with ffmpeg lambda layer.
- [ ] Bugfix
- [ ] Improvement
- [x] New feature
- [ ] Refactor
- [ ] Infrastructure
- [ ] Other (please specify)

## How would you describe the changes in a changelog (1-2 lines)?
Added ffmpeg layer to lambda, compress file when mp4, webm file is uploaded.
## How to test

- [ ] Needs regression testing (add QA needed label)

## Deploy risk & follow up

Please assess risk & follow up on deployment of this PR. Elaborate your choice if selecting anything other than 'No risk'.

- [x] No risk
- [ ] Low
- [ ] High

## Reminder list

- Determine whether REST API Documentation should be updated
- Make an assessment on whether unit tests should be updated and/or added
- Ensure follow up tasks are linked to this PR

QA - if regression testing was requested:

- [ ] QA has verified

### CODE REVIEW

File: handler.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 s3_client = boto3.client('s3',   config=my_config)
 
 def lambda_function(event, context):
+    print(event)
```
Comment (mhazad) [inline handler.py:15]: What is the purpose of logging the full event?

**Review by mhazad — CHANGES_REQUESTED** [review]
Please address the feedback.

File: handler.py (hunk 1 above)
Comment (mukutdanielroy) [reply handler.py:15]: Please accept my apologies. It was not meant to be pushed.

**Review by mhazad — APPROVED** [review]
LGTM

**Pull request merged**